src/taco-utils/tacoError.ts

```diff
@@ -76,7 +76,7 @@ module TacoUtility {
                     message = resources.getString(errorToken, args);
                 }
             } else {
```

Comment:
Do you know if this conditional is necessary? this function was just `return resources.getString(errorToken, args)` does anything misbehave? I thought that the ResourcesManager could deal with an empty array, but I wasn't sure.